### ISSUE TITLE
NuGet update fails due to TypeInitializationException

### ISSUE THREAD

**Issue by martincostello** [dependabot/dependabot-core#9245]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package ecosystem

NuGet

### Manifest location and content before the Dependabot update

[Directory.Packages.props](https://github.com/martincostello/browserstack-automate/blob/5067362051e9f68ac53f12a6330610982dc10ed2/Directory.Packages.props)

### dependabot.yml content

[.github/dependabot.yml](https://github.com/martincostello/browserstack-automate/blob/5067362051e9f68ac53f12a6330610982dc10ed2/.github/dependabot.yml)

### Updated dependency

Fails to update `Microsoft.AspNetCore.WebUtilities`.

### What you expected to see, versus what you actually saw

Expected Dependabot to not error.

Instead the run fails with the following errors:

```
updater | 2024/03/08 05:06:29 INFO <job_797491692> Updating Microsoft.AspNetCore.WebUtilities from 2.1.1 to 2.2.0
updater | running NuGet updater:
updater | /opt/nuget/NuGetUpdater/NuGetUpdater.Cli update --repo-root /home/dependabot/dependabot-updater/repo --solution-or-project /home/dependabot/dependabot-updater/repo/src/MartinCostello.BrowserStack.Automate/MartinCostello.BrowserStack.Automate.csproj --dependency Microsoft.AspNetCore.WebUtilities --new-version 2.2.0 --previous-version 2.1.1 --verbose
updater | 2024/03/08 05:06:30 ERROR <job_797491692> Block argument of NuGetConfigCredentialHelpers::patch_nuget_config_for_action causes an exception A compatible .NET SDK was not found.
updater | 
updater | Requested SDK version: 8.0.201
updater | global.json file: /home/dependabot/dependabot-updater/repo/global.json
updater | 
updater | Installed SDKs:
updater | 
updater | Install the [8.0.201] .NET SDK or update [/home/dependabot/dependabot-updater/repo/global.json] to match an installed SDK.
updater | 
updater | Learn about SDK resolution:
updater | https://aka.ms/dotnet/sdk-not-found
updater | Unhandled exception: System.TypeInitializationException: The type initializer for 'NuGetUpdater.Core.MSBuildHelper' threw an exception.
updater |  ---> System.InvalidOperationException: Failed to find all versions of .NET Core MSBuild. Call to hostfxr_resolve_sdk2. There may be more details in stderr.
updater |    at Microsoft.Build.Locator.DotNetSdkLocationHelper.GetSdkFromGlobalSettings(String workingDirectory)
updater |    at Microsoft.Build.Locator.DotNetSdkLocationHelper.GetDotNetBasePaths(String workingDirectory)+MoveNext()
updater |    at Microsoft.Build.Locator.DotNetSdkLocationHelper.GetInstances(String workingDirectory)+MoveNext()
updater |    at Microsoft.Build.Locator.MSBuildLocator.GetInstances(VisualStudioInstanceQueryOptions options)+MoveNext()
updater |    at System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
updater |    at System.Linq.Enumerable.TryGetFirst[TSource](IEnumerable`1 source, Boolean& found)
updater |    at System.Linq.Enumerable.First[TSource](IEnumerable`1 source)
updater |    at NuGetUpdater.Core.MSBuildHelper.RegisterMSBuild() in /opt/nuget/lib/NuGetUpdater/NuGetUpdater.Core/Utilities/MSBuildHelper.cs:line 42
updater |    at NuGetUpdater.Core.MSBuildHelper..cctor() in /opt/nuget/lib/NuGetUpdater/NuGetUpdater.Core/Utilities/MSBuildHelper.cs:line 34
updater |    --- End of inner exception stack trace ---
updater |    at NuGetUpdater.Core.MSBuildHelper.RegisterMSBuild() in /opt/nuget/lib/NuGetUpdater/NuGetUpdater.Core/Utilities/MSBuildHelper.cs:line 40
updater |    at NuGetUpdater.Core.UpdaterWorker.RunAsync(String repoRootPath, String workspacePath, String dependencyName, String previousDependencyVersion, String newDependencyVersion, Boolean isTransitive) in /opt/nuget/lib/NuGetUpdater/NuGetUpdater.Core/Updater/UpdaterWorker.cs:line 22
updater |    at NuGetUpdater.Cli.Commands.UpdateCommand.<>c__DisplayClass7_0.<<GetCommand>b__0>d.MoveNext() in /opt/nuget/lib/NuGetUpdater/NuGetUpdater.Cli/Commands/UpdateCommand.cs:line 37
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Invocation.AnonymousCommandHandler.InvokeAsync(InvocationContext context)
updater |    at System.CommandLine.Invocation.InvocationPipeline.<>c__DisplayClass4_0.<<BuildInvocationChain>b__0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass17_0.<<UseParseErrorReporting>b__0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass12_0.<<UseHelp>b__0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass22_0.<<UseVersionOption>b__0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass19_0.<<UseTypoCorrections>b__0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c.<<UseSuggestDirective>b__18_0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass16_0.<<UseParseDirective>b__0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c.<<RegisterWithDotnetSuggest>b__5_0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass8_0.<<UseExceptionHandler>b__0>d.MoveNext()
updater | 8.0.100 [/usr/local/dotnet/current/sdk]:
updater | A compatible .NET SDK was not found.
updater | 
updater | Requested SDK version: 8.0.201
updater | global.json file: /home/dependabot/dependabot-updater/repo/global.json
updater | 
updater | Installed SDKs:
updater | 
updater | Install the [8.0.201] .NET SDK or update [/home/dependabot/dependabot-updater/repo/global.json] to match an installed SDK.
updater | 
updater | Learn about SDK resolution:
updater | https://aka.ms/dotnet/sdk-not-found
updater | Unhandled exception: System.TypeInitializationException: The type initializer for 'NuGetUpdater.Core.MSBuildHelper' threw an exception.
updater |  ---> System.InvalidOperationException: Failed to find all versions of .NET Core MSBuild. Call to hostfxr_resolve_sdk2. There may be more details in stderr.
updater |    at Microsoft.Build.Locator.DotNetSdkLocationHelper.GetSdkFromGlobalSettings(String workingDirectory)
updater |    at Microsoft.Build.Locator.DotNetSdkLocationHelper.GetDotNetBasePaths(String workingDirectory)+MoveNext()
updater |    at Microsoft.Build.Locator.DotNetSdkLocationHelper.GetInstances(String workingDirectory)+MoveNext()
updater |    at Microsoft.Build.Locator.MSBuildLocator.GetInstances(VisualStudioInstanceQueryOptions options)+MoveNext()
updater |    at System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
updater |    at System.Linq.Enumerable.TryGetFirst[TSource](IEnumerable`1 source, Boolean& found)
updater |    at System.Linq.Enumerable.First[TSource](IEnumerable`1 source)
updater |    at NuGetUpdater.Core.MSBuildHelper.RegisterMSBuild() in /opt/nuget/lib/NuGetUpdater/NuGetUpdater.Core/Utilities/MSBuildHelper.cs:line 42
updater |    at NuGetUpdater.Core.MSBuildHelper..cctor() in /opt/nuget/lib/NuGetUpdater/NuGetUpdater.Core/Utilities/MSBuildHelper.cs:line 34
updater |    --- End of inner exception stack trace ---
updater |    at NuGetUpdater.Core.MSBuildHelper.RegisterMSBuild() in /opt/nuget/lib/NuGetUpdater/NuGetUpdater.Core/Utilities/MSBuildHelper.cs:line 40
updater |    at NuGetUpdater.Core.UpdaterWorker.RunAsync(String repoRootPath, String workspacePath, String dependencyName, String previousDependencyVersion, String newDependencyVersion, Boolean isTransitive) in /opt/nuget/lib/NuGetUpdater/NuGetUpdater.Core/Updater/UpdaterWorker.cs:line 22
updater |    at NuGetUpdater.Cli.Commands.UpdateCommand.<>c__DisplayClass7_0.<<GetCommand>b__0>d.MoveNext() in /opt/nuget/lib/NuGetUpdater/NuGetUpdater.Cli/Commands/UpdateCommand.cs:line 37
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Invocation.AnonymousCommandHandler.InvokeAsync(InvocationContext context)
updater |    at System.CommandLine.Invocation.InvocationPipeline.<>c__DisplayClass4_0.<<BuildInvocationChain>b__0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass17_0.<<UseParseErrorReporting>b__0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass12_0.<<UseHelp>b__0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass22_0.<<UseVersionOption>b__0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass19_0.<<UseTypoCorrections>b__0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c.<<UseSuggestDirective>b__18_0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass16_0.<<UseParseDirective>b__0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c.<<RegisterWithDotnetSuggest>b__5_0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass8_0.<<UseExceptionHandler>b__0>d.MoveNext()
updater | 8.0.100 [/usr/local/dotnet/current/sdk]
updater | 
updater | running NuGet updater:
updater | /opt/nuget/NuGetUpdater/NuGetUpdater.Cli update --repo-root /home/dependabot/dependabot-updater/repo --solution-or-project /home/dependabot/dependabot-updater/repo/tests/MartinCostello.BrowserStack.Automate.Tests/MartinCostello.BrowserStack.Automate.Tests.csproj --dependency Microsoft.AspNetCore.WebUtilities --new-version 2.2.0 --previous-version 2.1.1 --verbose
updater | 2024/03/08 05:06:30 ERROR <job_797491692> Block argument of NuGetConfigCredentialHelpers::patch_nuget_config_for_action causes an exception A compatible .NET SDK was not found.
updater | 
updater | Requested SDK version: 8.0.201
updater | global.json file: /home/dependabot/dependabot-updater/repo/global.json
updater | 
updater | Installed SDKs:
updater | 
updater | Install the [8.0.201] .NET SDK or update [/home/dependabot/dependabot-updater/repo/global.json] to match an installed SDK.
updater | 
updater | Learn about SDK resolution:
updater | https://aka.ms/dotnet/sdk-not-found
updater | Unhandled exception: System.TypeInitializationException: The type initializer for 'NuGetUpdater.Core.MSBuildHelper' threw an exception.
updater |  ---> System.InvalidOperationException: Failed to find all versions of .NET Core MSBuild. Call to hostfxr_resolve_sdk2. There may be more details in stderr.
updater |    at Microsoft.Build.Locator.DotNetSdkLocationHelper.GetSdkFromGlobalSettings(String workingDirectory)
updater |    at Microsoft.Build.Locator.DotNetSdkLocationHelper.GetDotNetBasePaths(String workingDirectory)+MoveNext()
updater |    at Microsoft.Build.Locator.DotNetSdkLocationHelper.GetInstances(String workingDirectory)+MoveNext()
updater |    at Microsoft.Build.Locator.MSBuildLocator.GetInstances(VisualStudioInstanceQueryOptions options)+MoveNext()
updater |    at System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
updater |    at System.Linq.Enumerable.TryGetFirst[TSource](IEnumerable`1 source, Boolean& found)
updater |    at System.Linq.Enumerable.First[TSource](IEnumerable`1 source)
updater |    at NuGetUpdater.Core.MSBuildHelper.RegisterMSBuild() in /opt/nuget/lib/NuGetUpdater/NuGetUpdater.Core/Utilities/MSBuildHelper.cs:line 42
updater |    at NuGetUpdater.Core.MSBuildHelper..cctor() in /opt/nuget/lib/NuGetUpdater/NuGetUpdater.Core/Utilities/MSBuildHelper.cs:line 34
updater |    --- End of inner exception stack trace ---
updater |    at NuGetUpdater.Core.MSBuildHelper.RegisterMSBuild() in /opt/nuget/lib/NuGetUpdater/NuGetUpdater.Core/Utilities/MSBuildHelper.cs:line 40
updater |    at NuGetUpdater.Core.UpdaterWorker.RunAsync(String repoRootPath, String workspacePath, String dependencyName, String previousDependencyVersion, String newDependencyVersion, Boolean isTransitive) in /opt/nuget/lib/NuGetUpdater/NuGetUpdater.Core/Updater/UpdaterWorker.cs:line 22
updater |    at NuGetUpdater.Cli.Commands.UpdateCommand.<>c__DisplayClass7_0.<<GetCommand>b__0>d.MoveNext() in /opt/nuget/lib/NuGetUpdater/NuGetUpdater.Cli/Commands/UpdateCommand.cs:line 37
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Invocation.AnonymousCommandHandler.InvokeAsync(InvocationContext context)
updater |    at System.CommandLine.Invocation.InvocationPipeline.<>c__DisplayClass4_0.<<BuildInvocationChain>b__0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass17_0.<<UseParseErrorReporting>b__0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass12_0.<<UseHelp>b__0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass22_0.<<UseVersionOption>b__0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass19_0.<<UseTypoCorrections>b__0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c.<<UseSuggestDirective>b__18_0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass16_0.<<UseParseDirective>b__0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c.<<RegisterWithDotnetSuggest>b__5_0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass8_0.<<UseExceptionHandler>b__0>d.MoveNext()
updater | 8.0.100 [/usr/local/dotnet/current/sdk]:
updater | A compatible .NET SDK was not found.
updater | 
updater | Requested SDK version: 8.0.201
updater | global.json file: /home/dependabot/dependabot-updater/repo/global.json
updater | 
updater | Installed SDKs:
updater | 
updater | Install the [8.0.201] .NET SDK or update [/home/dependabot/dependabot-updater/repo/global.json] to match an installed SDK.
updater | 
updater | Learn about SDK resolution:
updater | https://aka.ms/dotnet/sdk-not-found
updater | Unhandled exception: System.TypeInitializationException: The type initializer for 'NuGetUpdater.Core.MSBuildHelper' threw an exception.
updater |  ---> System.InvalidOperationException: Failed to find all versions of .NET Core MSBuild. Call to hostfxr_resolve_sdk2. There may be more details in stderr.
updater |    at Microsoft.Build.Locator.DotNetSdkLocationHelper.GetSdkFromGlobalSettings(String workingDirectory)
updater |    at Microsoft.Build.Locator.DotNetSdkLocationHelper.GetDotNetBasePaths(String workingDirectory)+MoveNext()
updater |    at Microsoft.Build.Locator.DotNetSdkLocationHelper.GetInstances(String workingDirectory)+MoveNext()
updater |    at Microsoft.Build.Locator.MSBuildLocator.GetInstances(VisualStudioInstanceQueryOptions options)+MoveNext()
updater |    at System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
updater |    at System.Linq.Enumerable.TryGetFirst[TSource](IEnumerable`1 source, Boolean& found)
updater |    at System.Linq.Enumerable.First[TSource](IEnumerable`1 source)
updater |    at NuGetUpdater.Core.MSBuildHelper.RegisterMSBuild() in /opt/nuget/lib/NuGetUpdater/NuGetUpdater.Core/Utilities/MSBuildHelper.cs:line 42
updater |    at NuGetUpdater.Core.MSBuildHelper..cctor() in /opt/nuget/lib/NuGetUpdater/NuGetUpdater.Core/Utilities/MSBuildHelper.cs:line 34
updater |    --- End of inner exception stack trace ---
updater |    at NuGetUpdater.Core.MSBuildHelper.RegisterMSBuild() in /opt/nuget/lib/NuGetUpdater/NuGetUpdater.Core/Utilities/MSBuildHelper.cs:line 40
updater |    at NuGetUpdater.Core.UpdaterWorker.RunAsync(String repoRootPath, String workspacePath, String dependencyName, String previousDependencyVersion, String newDependencyVersion, Boolean isTransitive) in /opt/nuget/lib/NuGetUpdater/NuGetUpdater.Core/Updater/UpdaterWorker.cs:line 22
updater |    at NuGetUpdater.Cli.Commands.UpdateCommand.<>c__DisplayClass7_0.<<GetCommand>b__0>d.MoveNext() in /opt/nuget/lib/NuGetUpdater/NuGetUpdater.Cli/Commands/UpdateCommand.cs:line 37
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Invocation.AnonymousCommandHandler.InvokeAsync(InvocationContext context)
updater |    at System.CommandLine.Invocation.InvocationPipeline.<>c__DisplayClass4_0.<<BuildInvocationChain>b__0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass17_0.<<UseParseErrorReporting>b__0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass12_0.<<UseHelp>b__0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass22_0.<<UseVersionOption>b__0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass19_0.<<UseTypoCorrections>b__0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c.<<UseSuggestDirective>b__18_0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass16_0.<<UseParseDirective>b__0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c.<<RegisterWithDotnetSuggest>b__5_0>d.MoveNext()
updater | --- End of stack trace from previous location ---
updater |    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass8_0.<<UseExceptionHandler>b__0>d.MoveNext()
updater | 8.0.100 [/usr/local/dotnet/current/sdk]
updater | 
updater | 2024/03/08 05:06:31 ERROR <job_797491692> Error processing Microsoft.AspNetCore.WebUtilities (Dependabot::DependabotError)
updater | 2024/03/08 05:06:31 ERROR <job_797491692> FileUpdater failed
updater | 2024/03/08 05:06:31 ERROR <job_797491692> /home/dependabot/dependabot-updater/lib/dependabot/dependency_change_builder.rb:69:in `run'
updater | 2024/03/08 05:06:31 ERROR <job_797491692> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/sorbet-runtime-0.5.11193/lib/types/private/methods/call_validation.rb:272:in `bind_call'
updater | 2024/03/08 05:06:31 ERROR <job_797491692> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/sorbet-runtime-0.5.11193/lib/types/private/methods/call_validation.rb:272:in `validate_call'
updater | 2024/03/08 05:06:31 ERROR <job_797491692> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/sorbet-runtime-0.5.11193/lib/types/private/methods/_methods.rb:272:in `block in _on_method_added'
updater | 2024/03/08 05:06:31 ERROR <job_797491692> /home/dependabot/dependabot-updater/lib/dependabot/dependency_change_builder.rb:42:in `create_from'
updater | 2024/03/08 05:06:31 ERROR <job_797491692> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/sorbet-runtime-0.5.11193/lib/types/private/methods/call_validation.rb:272:in `bind_call'
updater | 2024/03/08 05:06:31 ERROR <job_797491692> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/sorbet-runtime-0.5.11193/lib/types/private/methods/call_validation.rb:272:in `validate_call'
updater | 2024/03/08 05:06:31 ERROR <job_797491692> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/sorbet-runtime-0.5.11193/lib/types/private/methods/_methods.rb:272:in `block in _on_method_added'
updater | 2024/03/08 05:06:31 ERROR <job_797491692> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/update_all_versions.rb:132:in `check_and_create_pull_request'
updater | 2024/03/08 05:06:31 ERROR <job_797491692> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/update_all_versions.rb:64:in `check_and_create_pr_with_error_handling'
updater | 2024/03/08 05:06:31 ERROR <job_797491692> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/update_all_versions.rb:39:in `block in perform'
updater | 2024/03/08 05:06:31 ERROR <job_797491692> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/update_all_versions.rb:39:in `each'
updater | 2024/03/08 05:06:31 ERROR <job_797491692> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/update_all_versions.rb:39:in `perform'
updater | 2024/03/08 05:06:31 ERROR <job_797491692> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/group_update_all_versions.rb:139:in `run_ungrouped_dependency_updates'
updater | 2024/03/08 05:06:31 ERROR <job_797491692> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/group_update_all_versions.rb:73:in `perform'
updater | 2024/03/08 05:06:31 ERROR <job_797491692> /home/dependabot/dependabot-updater/lib/dependabot/updater.rb:45:in `run'
updater | 2024/03/08 05:06:31 ERROR <job_797491692> /home/dependabot/dependabot-updater/lib/dependabot/update_files_command.rb:43:in `block in perform_job'
updater | 2024/03/08 05:06:31 ERROR <job_797491692> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/opentelemetry-api-1.2.3/lib/opentelemetry/trace/tracer.rb:37:in `block in in_span'
updater | 2024/03/08 05:06:31 ERROR <job_797491692> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/opentelemetry-api-1.2.3/lib/opentelemetry/trace.rb:70:in `block in with_span'
updater | 2024/03/08 05:06:31 ERROR <job_797491692> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/opentelemetry-api-1.2.3/lib/opentelemetry/context.rb:87:in `with_value'
updater | 2024/03/08 05:06:31 ERROR <job_797491692> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/opentelemetry-api-1.2.3/lib/opentelemetry/trace.rb:70:in `with_span'
updater | 2024/03/08 05:06:31 ERROR <job_797491692> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/opentelemetry-api-1.2.3/lib/opentelemetry/trace/tracer.rb:37:in `in_span'
updater | 2024/03/08 05:06:31 ERROR <job_797491692> /home/dependabot/dependabot-updater/lib/dependabot/update_files_command.rb:17:in `perform_job'
updater | 2024/03/08 05:06:31 ERROR <job_797491692> /home/dependabot/dependabot-updater/lib/dependabot/base_command.rb:36:in `run'
updater | 2024/03/08 05:06:31 ERROR <job_797491692> bin/update_files.rb:24:in `<main>'
```

**Comment by abdulapopoola** [comment]
This should be fixed in #9249

**Comment by martincostello** [comment]
Confirmed this specific issue is fixed.
